locales/base/translation.json

```diff
@@ -1137,6 +1137,7 @@
   "sent": "Sent",
   "received": "Received",
   "viewAll": "View All",
+  "youAreAllCaughtUp": "You're all caught up!",
```

Comment:
```suggestion
  "noNotificationsPlaceholder": "You're all caught up!",
```

Reply:
it's easier for the translators to translate accurately if the keys are named semantically :)